pom.xml

```diff
@@ -45,6 +45,7 @@
     <maven-source-plugin.version>2.4</maven-source-plugin.version>
     <maven-javadoc-plugin.version>2.10.3</maven-javadoc-plugin.version>
     <maven-gpg-plugin.version>1.6</maven-gpg-plugin.version>
+    <okhttp.version>3.2.0</okhttp.version>
```

Comment:
I would be in favor of keeping okhttp version and dependency in brave-okhttp/pom.xml. wdyt?